### PR TITLE
Feat/various updates

### PR DESCRIPTION
**Pull request by klubiniecki**
Few bits and pieces to update our code here and there 😃

### CODE REVIEW

**Comment by VasaviGannena** [comment]
close pull request

**Comment by klubiniecki** [comment]
@VasaviGannena Why did you close my PR? What was your intention here? 🙂

**Comment by VasaviGannena** [comment]
Oh sorry, I though that ,I have send PR request by mistake,That why I ask to close that PR request.

**Comment by klubiniecki** [comment]
> Oh sorry, I though that ,I have send PR request by mistake,That why I ask to close that PR request.

That's ok. If you don't know what you're doing, always ask. You can always see who opened a PR on the PR page: 

Screen Shot 2021-03-02 at 5 14 16 pm

File: README.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 <p align="center">
   <h1 align="center"><a href="https://codeforaustralia.github.io/seechange/">SeeChange (aka. Civic Makers Climate-Change) Project</a></h1>
```
Comment (hqtan) [inline README.md]: Hi @klubiniecki ,

Thanks for the PR! Apologies for the tardy review! :bow: 

The changes look awesome :+1: 

One minor addition/suggestion: Can we change the url to website from `https://codeforaustralia.github.io/seechange/` to `https://seechange.codeforaustralia.org/` ?

**Review by hqtan — APPROVED** [review]
lgtm!

**Pull request merged**